mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg

```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
 above).
 ###########################################################################
 
+
+##############################PRICES FORMUALE##############################
+Shields
+Non-Serpent Shields
+n^2.1 n={50:100} 		- where n is the tier adjusted to a fit a min/max
+						of 50 for tier 1 (e.g. Wood), to 100 for BlackMetal
+Serpent Shield (rationale: mid tier with high risk potential)
+n^2.1 n={75} -  where n is adjusted to be between Iron and Silver
+
+
+Chest & Leggings
+n^2 n={40:100} 			- where n is the tier adjusted to a fit a min/max
+						of 40 for tier 0 (e.g. Rags), to 100 for BlackMetal
+
+Helmets
+n^1.9 n={50:100} 		- where n is the tier adjusted to a fit a min/max
+						of 40 for tier 0 (e.g. Rags), to 100 for BlackMetal
+
+Capes
+n^1.8 n={50,60,90,100} 	- where n is the tier adjusted to a fit a min/max
```

Comment:
given how little protection is provided by capes, I'd be fine lowering the costs even further.  Except wolf.  That seems good.

Reply:
Okay, can do